dask_sql/physical/rel/logical/filter.py

```diff
@@ -31,7 +32,7 @@ def filter_or_scalar(df: dd.DataFrame, filter_condition: Union[np.bool_, dd.Seri
 
     # In SQL, a NULL in a boolean is False on filtering
     filter_condition = filter_condition.fillna(False)
-    return df[filter_condition]
+    return attempt_predicate_pushdown(df[filter_condition])
```

Comment:
Ha just saw this - [in my review](https://github.com/dask-contrib/dask-sql/pull/433#pullrequestreview-911971136) I pointed out that it might make sense to have this controlled through Dask's config module, with a default set in our own config YAML